.github/actions/create_mrt/action.yml

```diff
@@ -4,10 +4,14 @@ inputs:
     description: "Mobify user email"
   mobify_api_key:
     description: "Mobify user API key"
+  cwd:
+    required: false
+    default: ${PWD}
 runs:
   using: composite
   steps:
     - name: Create MRT credentials file
+      working-directory: ${{ inputs.cwd }}
```

Comment:
I'm not sure if you need to pass in a default directory or does it use root if value of cwd is not passed in. Can you check that ?

Reply:
I added the root directory to support pwa-sfra-hybrid-test jobs.

Here in PWA, we don't need to set a cwd as the default directory is the root working directory for the job.

However, in pwa-sfra-hybrid-test, both PWA and plugin_slas are checked out into sub directories and I wanted the ability to set these actions to run in the relevant subdirectory (ie. PWA actions running in the pwa directory, plugin_slas actions running in the plugin_slas directory)